Updates/4.12.4-b3.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\TagManager;
+
+use Piwik\Plugins\TagManager\Template\Variable\MatomoConfigurationVariable;
+use Piwik\Plugins\TagManager\Updates\NewVariableParameterMigrator;
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 4.12.4-b1.
+ */
+class Updates_4_12_4_b3 extends PiwikUpdates
```

Comment:
@tomper00 Sorry but can you update this migration class name as per your other PR suggestion ?